api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -107,6 +110,20 @@ public void validate(Object target, Errors errors) {
 				validateStopDatetime(visit, otherVisit, errors);
 			}
 		}
+        	if(visit.getStartDatetime()!=null && visit.getPatient()!=null && visit.getPatient().getBirthdate()!=null)
+        	{
+            		Date birthDateTime = visit.getPatient().getBirthdate();
```

Comment:
Why don't you declare and assign these two before the above if check? That way you can reuse them even in the if check.